examples/stable/elasticsearch/README.md

```diff
@@ -91,12 +83,20 @@ health status index    uuid                   pri rep docs.count docs.deleted st
 green  open   customer xbwj34pTSZOdDI7xVR0qIA   5   1          1            0      8.9kb          4.4kb
 ```
 
+## Create the Blueprint 
+
+In order to perform backup, restore, and delete operations on the running elasticsearch, we need to create a blueprint. You can create that using the command below from the root of kanister repo.
+
+```bash
+kubectl create -f ./examples/helm/kanister/kanister-elasticsearch/kanister/elasticsearch-blueprint.yaml -n kasten-io
```

Comment:
```suggestion
kubectl create -f ./examples/stable/elasticsearch/elasticsearch-blueprint.yaml -n kasten-io
```

Reply:
Made the changes to reflect correct path.